src/pages/index.js

```diff
@@ -83,20 +71,65 @@ const editProfileFormValidator = new FormValidator(
   settings,
   editProfileModalFormElement
 );
+const editAvatarFormValidator = new FormValidator(
+  settings,
+  editAvatarModalFormElement
+);
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/cohort-3-en",
+  headers: {
+    authorization: "b685d3e0-616a-4dae-bc5b-53892a4f7953",
+    "Content-Type": "application/json",
+  },
+});
+function createCard(item) {
+  // create instance of Card class
+  const card = new Card(
+    item,
+    "#card-template",
+    onCardClick,
+    handleCardDeleteFunctionInIndexComponent
+  );
+  //create a card by calling getCardElement method from Card class
+  const cardElement = card.getCardElement();
+  //return the card
+  return cardElement;
+}
+
+api.getUserInfo().then((data) => {
+  const name = data.name;
+  const subtitle = data.about;
+  const url = data.avatar;
+  userInfo.setUserInfo({ name, subtitle });
+  userInfo.setNewAvatar(url);
+});
+
+api
+  .getInitialCards()
+  .then((data) => {
+    // process the result
+    data.forEach((item) => {
```

Comment:
You need to use `renderItems` for rending the array. You should just change it so that it could accept the array into the call


```js
  renderItems(items) {
    items.forEach((cardData) => {
      const cardItem = this._renderer(cardData);
      this.appendItem(cardItem);
    });
  }

```